doc/pipeline_details.rst

```diff
@@ -47,9 +48,12 @@ Read the description for more information.
 |                       |                                      | takes to receive a response the longer loads and stores     |
 |                       |                                      | will stall.                                                 |
 +-----------------------+--------------------------------------+-------------------------------------------------------------+
-| Multiplication        | 2/3 (Fast Multiplier)                | Fast: 2 for MUL, 3 for MULH.                                |
-|                       |                                      | Slow: clog2(``op_b``) for MUL, 32 for MULH.                 |
-|                       | clog2(``op_b``)/32 (Slow Multiplier) | See details in :ref:`mult-div`                              |
+| Multiplication        | 0/1 (Single-Cycle Multiplier         | 0 for MUL, 1 for MULH.                                      |
+|                       +--------------------------------------+-------------------------------------------------------------+
```

Comment:
Please remove the additional horizontal lines. I just does not render nicely.

Please just leave one "empty line" between to multiplier implementations/rows.

Reply:
Ok, done.

Reply:
thx